assets/src/dashboard/app/views/myStories/karma/myStories.karma.js

```diff
@@ -274,6 +278,9 @@ describe('Grid view', () => {
 
       await fixture.events.keyboard.type(firstStoryTitle);
 
+      // Wait for the debounce
+      await sleep(300);
```

Comment:
This is failing for me. 

`Expected 4 to equal 1.`

Reply:
eh, helps if you pull the latest of the branch before running tests. your test is fine! my b!